app/src/main/java/com/github/libretube/ui/fragments/DownloadsFragment.kt

```diff
@@ -168,12 +212,13 @@ class DownloadsFragment : DynamicLayoutManagerFragment() {
         super.onStart()
     }
 
+    @RequiresApi(Build.VERSION_CODES.TIRAMISU)
```

Comment:
That makes the fragment crash or behave wrong on older Android versions ...

Reply:
I have undone that change.